src/constellation/constellation.py

```diff
@@ -335,18 +335,39 @@ def remove(self):
         docker_util.remove_network(self.name)
 
 
-class ConstellationMount:
-    def __init__(self, name, path, **kwargs):
-        self.name = name
-        self.path = path
+# Base class for mounts.
+class _ConstellationMount:
```

Comment:
Make this an ABC, or even better a Protocol (sort of like a Java interface): https://typing.python.org/en/latest/spec/protocol.html

I don’t really find that the common constructor adds that much value. I’d just make each class self contained that happen to implement the same structural interface.